### PR TITLE
chore(cli): remove package-scripts alias

### PR DESCRIPTION
**Pull request by sxn**
**What**: remove `package-scripts` as an alias

**Why**: see #54 

**How**: just removing it from `package.json` and updating the README & tests.

This removes the 'package-scripts' alias as it conflicts with 'package-scripts.js' on Windows

issue: #54
breaking: 'package-scripts' won't be available as an alias anymore

### CODE REVIEW

**Comment by kentcdodds** [comment]
Looks great! What do you think about using `nps` rather than `p-s`? I like the `nps` alias better and want to encourage others to use it more.

**Comment by sxn** [comment]
The less I have to move my hands on the keyboard, the better, so I use `nps` too :smile:

**Comment by kentcdodds** [comment]
Yeah, let's have most of the documentation use `nps`. Maybe we can add a note about why at the beginning somewhere.

**Comment by sxn** [comment]
Done :) I left occurrences of `p-s` when they were referring to the package itself not the CLI, but I might have missed some/also changed some cases that I shouldn't've. Can someone have a look please?

File: README.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ which has fundamental issues (like no comments).
 
 ## This solution
 
-Put all of your scripts in a file called `package-scripts.js` and use `p-s` in a single `package.json` script:
+Put all of your scripts in a file called `package-scripts.js` and use `nps` in a single `package.json` script:
```
Comment (kentcdodds) [inline README.md]: Let's mention that `nps` is the recommended alias right here. Otherwise people may be confused.

File: README.md [hunk 2]
```diff
@@ -32,14 +32,18 @@ which has fundamental issues (like no comments).
...
+
+Note: We used `nps` even though the package we just installed is named `p-s`. This is because internally we use `nps`
```
Comment (kentcdodds) [inline README.md]: Could we note that `nps` is an acronym for `npm package scripts`?

**Comment by kentcdodds** [comment]
LGTM

**Comment by kentcdodds** [comment]
TIL why it's important to keep PRs up-to-date with `master` before merging them :)

screen shot 2016-09-12 at 11 48 07 am

**Comment by sxn** [comment]
Thanks! Sorry for not answering, I'm currently on holiday so I'm only near my computer in the morning or evening, and coupled with the timezone difference it can get .. odd. :crying_cat_face:

**Comment by kentcdodds** [comment]
No worries!

**Pull request merged**